i18n/en.json

```diff
@@ -371,6 +371,10 @@
     "id": "api.channel.update_channel.deleted.app_error",
     "translation": "The channel has been archived or deleted"
   },
+  {
+    "id": "api.channel.update_channel.invalid_character.app_error",
+    "translation": "Invalid channel name for non-direct channels. Cannot have user id."
```

Comment:
Recommend updating to be more clear and consistent in identifying the channel as non-direct **message** channel:  

"Invalid channel name.  User ids are not permitted in channel name for non-direct message channels."